lib/index.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express');
+
+const { SessionNotFound } = require('./errors');
+const { startSession, stopSession } = require('./sessions');
+
+const port = 3000;
+
+const app = express();
+
+app.use(express.json());
+
+app.post('/har-sessions', async (req, res, next) => {
+  const { browserURL } = req.body;
+  if (!browserURL) {
+    res.status(400).json({ error: "Missing browserURL" });
+    return;
+  }
+
+  try {
+    const sessionId = await startSession(browserURL);
+    res.status(201).json({ sessionId: sessionId.toString() });
+  } catch (e) {
+    next(e);
```

Comment:
For my own edification, can you explain what `next` does here (and on line 37 in the `DELETE` route/handler)? From what I can gather. it's supposed to pass execution to the next matching route but it doesn't look like there are any?

Reply:
FWIW I'm not an Express.js expert, but from what I can tell, there's an implicit, default middleware that'll handle errors if you pass them via `next`.

See https://expressjs.com/en/guide/error-handling.html#the-default-error-handler for some more details. I added some tests, too, to make sure this code works as expected.